config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -19,9 +19,12 @@ presubmits:
         securityContext:
           privileged: true
         resources:
+          limits:
+            cpu: 7300m
+            memory: "36Gi"
           requests:
-            cpu: 7
-            memory: "41Gi"
```

Comment:
so this is lowering the request? I think the job runs differently on a 40GB memory boundary (serial vs parallel)

https://github.com/kubernetes/kubernetes/blob/aa0632208ec0a2191e956d86f3406afbb497b881/hack/lib/golang.sh#L307-L310

not sure what we want here

Reply:
I’m ok if we keep it at 41 for now. It looked like it never hit that peak but we can find out later once we move it off k8s-prow-builds

Reply:
done @spiffxp